extensions/dbgobject/type/dbgobject-type.js

```diff
@@ -26,7 +26,8 @@ Loader.OnLoad(function () {
 
         // Normalize type name.
         this._name = name.trim();  // strip outer whitespace
-        this._name = this._name.replace(/^const\s+/, "");  // remove leading "const" and whitespace
+        this._name = this._name.replace(/&/g, "*"); // treat references like pointers
+        this._name = this._name.replace(/^((const|volatile)\s+)+/g, "");  // remove leading "const" and whitespace
```

Comment:
Should we also handle "mutable" here?

Reply:
Yes, the last + in the regex and the /g handle that case.

Reply:
Hmm, GDB does not consider "mutable" to be part of the type. And I guess it isn't really part of the type -- more part of the field declaration? So I think it's fine to leave as-is.

Reply:
Ok, good to know. This is fine in that case.